manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -126,6 +126,26 @@ spec:
           - name: "REPAIR_BROKEN-POD-LABEL-VALUE"
             value: "{{.Values.cni.repair.brokenPodLabelValue}}"
 {{- end }}
+
+{{- if .Values.cni.taint.enabled }}
```

Comment:
Again, I **strongly** recommend not making a new container. I cannot emphasize this enough

Reply:
I agree with this sentiment. Why did we have to add a new container for this?

It feels a bit off, running as many of instances of the taint controller as that of the node count